test/e2e/tests/account/snap-account-transfers.spec.ts

```diff
@@ -33,6 +34,12 @@ describe('Snap Account Transfers @no-mmi', function (this: Suite) {
         ganacheServer?: Ganache;
       }) => {
         await loginWithBalanceValidation(driver, ganacheServer);
+
+        await tempToggleSettingRedesignedTransactionConfirmations(driver);
```

Comment:
Do we not want to test these flows on the redesign as well?

Reply:
I originally categorised these tests as "User flows that we don't need to migrate immediately, but will have to be migrated or adapted when old confirmations flows are deleted, 🟡", but I'm changing that and duplicating them right away.